src/handle-data.ts

```diff
@@ -1,6 +1,10 @@
+import { stardust } from '@nebula.js/stardust';
+import { TableLayout, HyperCube, ExtendedNxDimensionInfo, ExtendedNxMeasureInfo, Column, PageInfo } from './types';
+
 const directionMap = {
   A: 'asc',
   D: 'desc',
+  N: 'asc',
```

Comment:
why is this needed?

Reply:
good question. Becasue of the value of`info.qSortIndicator` in `directionMap[info.qSortIndicator]` is A or D or N. https://github.com/qlik-oss/sn-table/pull/586/files#diff-26fa10f48243e01a30f0b2eee16fa394336656c6e83d68066718bad1d1ccb1cdR55